values.yaml

```diff
@@ -212,8 +212,13 @@ st2web:
       cpu: "50m"
     limits:
       memory: "100Mi"
-  # TODO: Add Ingress and/or LoadBalancer setting as a way to expose service to public (#6)
+  # TODO: Add Ingress setting as a way to expose service to public (#6).
+  # For now, use LoadBalancer or NodePort (default to LoadBalancer).
   # ingress:
+  service:
```

Comment:
`service` will need to expose in values.yaml optional `annotation` field to configure additional settings is it SSL or AWS-load-balancer specific or something else

See an example: https://kubernetes.io/docs/concepts/services-networking/service/#ssl-support-on-aws